### ISSUE TITLE
Поддержка внешних CLI декодеров

### ISSUE THREAD

**Issue by greenvorbisfan** [Alexey-Yakovenko/deadbeef#1725]
Возможно ли реализовать поддержку внешних декодеров наподобие того, как это сделано в foobar2000? Есть отдельные форматы (например, La: [http://www.lossless-audio.com/download.htm](url)), которые не поддерживает даже ffmpeg.
Ссылка на плагин foobar2000: [http://www.foobar2000.org/components/view/foo_input_exe](url)
Как это выглядит:
![foo_wrapper](https://cloud.githubusercontent.com/assets/7892693/23947141/7eae61fa-098e-11e7-9ca0-b1eda1c3cd14.png)

**Comment by Alexey-Yakovenko** [comment]
Yes, it is technically possible in exactly the same way as on windows / fb2k.
